docker-compose.debug.yml

```diff
@@ -20,6 +20,35 @@ services:
       - ./dist:/build/
     networks:
       - debug
+  mandatees-decisions:
+    image: lblod/delta-consumer:0.0.24
+    volumes:
+      - ./config/custom-dispatching:/config/triples-dispatching/custom-dispatching
+      - ./data/files/mandatees-decisions:/consumer-files
+    environment:
+      DCR_SERVICE_NAME: 'mandatees-decisions'
+      DCR_SYNC_BASE_URL: 'http://producer-identifier'
+      DCR_JOB_CREATOR_URI: 'http://lblod.data.gift/services/delta-producer-background-jobs-initiator-mandatees-decisions'
+      DCR_SYNC_DATASET_SUBJECT: 'http://data.lblod.info/datasets/delta-producer/dumps/Mandatees-DecisionsCacheGraphDump'
+      DCR_INITIAL_SYNC_JOB_OPERATION: 'http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/initialPublicationGraphSyncing/mandatees-decisions'
+      DCR_DELTA_SYNC_JOB_OPERATION: 'http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/deltaDumpFileCreation/mandatees-decisions'
+      DCR_START_FROM_DELTA_TIMESTAMP: '2024-01-12T15:51:23.478Z'
+      DCR_SYNC_FILES_PATH: '/sync/mandatees-decisions/files'
+      DCR_KEEP_DELTA_FILES: true
+      DCR_DELTA_FILE_FOLDER: '/deltas/mandatees-decisions/files'
+      DCR_CRON_PATTERN_DELTA_SYNC: '0 * * * *	' # Every hour
```

Comment:
Update this in your override file to be  `DCR_CRON_PATTERN_DELTA_SYNC: '0/15 * * * * *' # Every 15 seconds minute`